features/fixtures/mazerunner/src/main/java/com/bugsnag/android/mazerunner/scenarios/AutoSessionSmokeScenario.kt

```diff
@@ -0,0 +1,27 @@
+package com.bugsnag.android.mazerunner.scenarios
+
+import android.content.Context
+import android.content.Intent
+import android.os.Handler
+import android.os.Looper
+import com.bugsnag.android.Bugsnag
+import com.bugsnag.android.Configuration
+
+/**
+ * Sends an automated session payload to Bugsnag.
+ */
+internal class AutoSessionSmokeScenario(config: Configuration,
+                                        context: Context) : Scenario(config, context) {
+    override fun run() {
+        super.run()
+        config.autoTrackSessions = true
+        Bugsnag.start(context, config)
+        context.startActivity(Intent("com.bugsnag.android.mazerunner.UPDATE_CONTEXT"))
+
+        val main = Handler(Looper.getMainLooper())
+        main.postDelayed(Runnable {
```

Comment:
This feels like it could introduce a source of flakes - the Android system will need to launch the new activity in < 1s. Using the [activity lifecycle callbacks](https://developer.android.com/reference/android/app/Application.ActivityLifecycleCallbacks#onActivityCreated(android.app.Activity,%20android.os.Bundle)) instead to detect when the new activity has launched would make this predictable and avoid the arbitrary wait - a couple of other scenarios have examples of how to achieve this.